pytorch_lightning/metrics/classification/f_beta.py

```diff
@@ -124,9 +125,11 @@ def compute(self):
             precision = self.true_positives.sum().float() / (self.predicted_positives.sum() + METRIC_EPS)
```

Comment:
do we really want to keep our metric systematically unprecise by adding some offset?

Reply:
we can move towards `class_reduce` function which explicit handles `nans`, when we unify the class based metrics and functional metrics.

Reply:
yeah, makes sense. Just checked and the tests pass without METRIC_EPS. Pushing the update now.